src/api/schemas.ts

```diff
@@ -8,8 +8,24 @@ export const OpenApiSchemaOptions: SwaggerOptions = {
   openapi: {
     info: {
       title: 'Ordinals API',
-      description:
-        'A service that indexes Bitcoin Ordinals data and exposes it via REST API endpoints.',
+      description: `
+The [Ordinals API](https://docs.hiro.so/ordinals-api) is a service that indexes Bitcoin Ordinals data and exposes it via REST API endpoints.
+
+Here are the key features of the Ordinals API:
+
+- **Ordinal Inscription Ingestion**: The Ordinals API helps with the complete ingestion of ordinal inscriptions. Using our endpoints, you can retrieve the metadata for a particular inscription, all inscriptions held by a particular address, trading activity for inscriptions, and more.
+
+- **BRC-20 Support**: The Ordinals API also offers support for BRC-20 tokens, a fungible token standard built on top of ordinal theory. Retrieve data for a particular BRC-20 token, a user's BRC-20 holdings, marketplace activity, and more.
+
+- **REST JSON Endpoints with ETag Caching**: The Ordinals API provides easy-to-use REST endpoints that return responses in JSON format. It also supports ETag caching, which allows you to cache responses based on inscriptions. This helps optimize performance and reduce unnecessary requests.
+
+- **Auto-Scale Server Configurations**: The Ordinals API supports three run modes based on the RUN_MODE environment variable:
+  - \`default\`: This mode runs all background jobs and the API server. It is suitable for running a single instance of the API.
+  - \`readonly\`: Only the API server runs in this mode. It is designed for auto-scaled clusters with multiple \`readonly\` instances and a single \`writeonly\` instance. The \`writeonly\` instance is responsible for populating the database.
+  - \`writeonly\`: This mode is used in an auto-scaled environment to consume new inscriptions and push that data to a database. It works in conjunction with multiple \`readonly\` instances.
```

Comment:
I think this part is not relevant in an OpenAPI documentation because we're not in the context of a dev who wants to run the API, only a consumer cc @max-crawford

Reply:
Ah, that's a good call, we should remove this part